### ISSUE TITLE
Shape keys no longer transfer to Vroid models made in the newest version

### ISSUE THREAD

**Issue by Kryptos03** [hinzka/52blendshapes-for-VRoid-face#3]
Due to an increase in polycount on the facemesh in the newest version of vroid, the shape keys no longer transfer to newer models.

**Comment by hinzka** [comment]
I checked with VRoid Studio v1.4.2 (the latest version at this time).
I checked the Inspector of the Face mesh on Unity and the number of vertices and triangulated faces matched.
(UniVRM: v0.94.0)
The latest version of VRoid can copy Blendshapes without any problems.

What is the number of vertices and triangulated faces in the model where the problem is occurring?

![2022-03-22_12h03_44](https://user-images.githubusercontent.com/32916633/159400478-427b5b3c-b968-41fd-979b-1829c7f25ed2.png)


![VRoidSample_A](https://user-images.githubusercontent.com/32916633/159398843-da8e39e8-d589-4f28-b8fe-f3388a45a069.png)
![2022-03-22_11h54_24](https://user-images.githubusercontent.com/32916633/159398854-dabfd44c-85e6-4b42-9b05-15cd18ea7495.png)
